prow/test/integration/README.md

```diff
@@ -8,14 +8,35 @@ Just run:
 ./prow/test/integration/integration-test.sh
 ```
 
-This script [sets up](#setup) the environment, [runs](#run-test) all the available integration tests, and then [cleans up](#cleanup) everything.
+This script creates a [KIND](https://kind.sigs.k8s.io/) Kubernetes cluster, runs all available integration tests, and finally deletes the cluster.
+
+# How it works
```

Comment:
Move this below how to run and above how to add a new test. Most people reading the README will likely want to just know how to run the tests